### PR TITLE
Ethereals now get sick from non ethereal food 

### PR DESCRIPTION
**Pull request by KittyNoodle**
## About The Pull Request
Ethereals now find non-ethereal food to be toxic(note that toxic food does not actually do damage but causes disgust without needing to taste(ethereals have no tastebuds))

## Why It's Good For The Game

Lore accuracy good

## Changelog
:cl:
expansion: ethereals now actually dislike non-ethereal food
fix: jellypeople and fly people now actually find ethereal food toxic
refactor: ethereal food now has a new category 
/:cl:

### CODE REVIEW

File: code/__DEFINES/food.dm [hunk 1]
```diff
@@ -14,6 +14,7 @@
 #define BREAKFAST (1<<13)
 #define CLOTH (1<<14)
 #define NUTS (1<<15)
+#define ETHEREAL (1<<16) //i don't know what the fuck (1<<16) means but there is a pattern here so lets just go with that i guess
```
Comment (Wallemations) [inline code/__DEFINES/food.dm:17]: This comment is unnecessary.
And to answer your comment, that's just how flags are defined, and you were correct in following the pattern.

**Comment by wesoda25** [comment]
Rather lame imo, seeing as ethereal food is not very prevalent.

**Comment by EricZilla** [comment]
Unironically this really opens up the doors for an ethereal food PR

**Comment by Watermelon914** [comment]
Feature additions to species requires prior approval, please check the contributor guidelines.

**Comment by KittyNoodle** [comment]
Fuck, got permissions for the other one so I thought this would be okay, gonna ask now

**Pull request closed without merging**